### DISCUSSION TITLE
IRC channel

### DISCUSSION THREAD

**Discussion by Poorchop** [koreader/koreader#8642] [General]
Would the developers consider creating an official IRC channel? Between digging through issues, documentation, the wiki, and threads on various forums, I try to gather information pertinent to my queries but sometimes it's much easier when one can just propose a question in a channel. Some questions also seem too minor to warrant a discussion thread such as this one, whereas an IRC channel is a much more informal venue for simple questions.

**Comment by NiLuJe** [comment]
That's where [Gitter](https://gitter.im/koreader/koreader) comes in ;).

**Reply by Poorchop** [reply]
Thanks. I hadn't realized that there was a Gitter for this project.
